src/utils/DynamicContent/formatters/CurrencyDe.ts

```diff
@@ -23,8 +23,18 @@ export class CurrencyDe implements Currency {
 		return this._euroAmountFormatter( amount ).replace( ',00', '' );
 	}
 
-	public customAmountInput( amount: number ): string {
-		return this._customAmountInputFormatter( amount );
+	public euroAmountFromCents( amountInCents: number ): string {
+		const floatValue = Math.trunc( amountInCents ) / 100;
```

Comment:
The `Math.trunc` is for removing any accidental decimal places that linger in the cent amount?

Reply:
exactly, the Math.trunc() is used for silently getting rid of wrong (float) input values